orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/providers/aws/AwsDeployStagePreProcessor.groovy

```diff
@@ -156,8 +160,11 @@ class AwsDeployStagePreProcessor implements DeployStagePreProcessor {
   }
 
   private static boolean shouldCheckServerGroupsPreconditions(StageData stageData) {
-    // TODO(dreynaud): enabling cautiously for RRB only for testing, but we would ideally roll this out to other strategies
-    return (Strategy.fromStrategyKey(stageData.strategy) == Strategy.ROLLING_RED_BLACK) && (stageData.maxInitialAsgs != -1)
+    // TODO(dreynaud): enabling cautiously for RRB/MD only for testing, but we would ideally roll this out to other strategies
+    Strategy strategy = Strategy.fromStrategyKey(stageData.strategy)
+
+    return (((strategy == Strategy.ROLLING_RED_BLACK) || (strategy == Strategy.MONITORED))
```

Comment:
@dreynaud i know you prefer `in []`

Reply:
`supportedStrategies.contains(strategy)`? :)